core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.html

```diff
@@ -78,6 +78,31 @@ <h2>{{ variant.experimentName }}</h2>
 
 <ng-template #defaultHeaderRightTpl>
     <dot-global-message data-testId="globalMessage" right></dot-global-message>
+    <p-tag
```

Comment:
Add a <ng-container *ngIf.... to handle only one IF, the tag and the button wont be showed if dont have any experiment.

Reply:
just moved to one element not needed anymore.